reactor-pool/src/main/java/reactor/pool/AbstractPool.java

```diff
@@ -441,14 +441,18 @@ public void request(long n) {
 		 * Stop the countdown started when calling {@link AbstractPool#doAcquire(Borrower)}.
 		 */
 		void stopPendingCountdown(boolean success) {
-			if (!timeoutTask.isDisposed()) {
-				if (success) {
-					pool.metricsRecorder.recordPendingSuccessAndLatency(pool.clock.millis() - pendingAcquireStart);
-				} else {
-					pool.metricsRecorder.recordPendingFailureAndLatency(pool.clock.millis() - pendingAcquireStart);
-				}
+			if (timeoutTask == null) {
+				return;
 			}
+
+            if (success) {
+                pool.metricsRecorder.recordPendingSuccessAndLatency(pool.clock.millis() - pendingAcquireStart);
```

Comment:
When the timeout is not specified how/where is `pendingAcquireStart` initialised?

Reply:
Fix pendingAcquireStart to initialized when timeout is not specified